ci_framework/plugins/action/ci_script.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+
+# Copyright Red Hat, Inc.
+# Apache License Version 2.0 (see LICENSE
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+action: ci_script
+
+short_description: Runs a string given script remotly while providing logging
+
+description:
+- Discover latest image matching a specific pattern on remote server.
+
+options:
+    output_dir:
+        description: The path in where the script will be copied
+        required: true
+        type: str
+    cmd:
+        description: The shell script content to be run
+        required: true
+        type: str
+    debug:
+        description: Enables bash tracing in the script
+        required: true
```

Comment:
You apparently set a default to True later - though imho we shouldn't default to True, nor require that parameter, unless the CI Framework adds a new `ROLENAME_debug` parameter, with a global `cifmw_debug` and a default value to `False`.
Reason to default to `False` : in case of a command aiming to extract data, the `-x` will break the output, and create a really messy situation.
For such cases, we MAY want to be smarter with the stdout and, probably, split a bit things (i.e. using PS4 or something like that...)